test/integration/account_settings_test.go

```diff
@@ -0,0 +1,47 @@
+package integration
+
+import (
+    "context"
+    "testing"
+
+    "github.com/stretchr/testify/require"
+    "github.com/linode/linodego"
+)
+
+func TestAccountSettings_Get(t *testing.T) {
```

Comment:
Is there any chance this test could be updated to not require any pre-configuration of an account when running fixtures?

Reply:
we can use library like httpmock to intercept and mock HTTP requests made by your client
i will try to adopt it and raise PR again